debian/indi-sv305/changelog

```diff
@@ -1,3 +1,9 @@
+indi-sv305 (1.3.0) kakura; urgency=low
+
+  * SV405CC support added
+  * Mutex conflicts fix
+  * OFFSET support on Ekos Camera module
+
```

Comment:
You must add this line "-- Blaise-Florentin Collin <thx8411@yahoo.fr>  Thu, 24 Feb 2022 16:00:00 +0300", with updated author and date, or the package will be broken.

Debain the package changelog file is very strict. See https://www.debian.org/doc/debian-policy/ch-source.html#s-dpkgchangelog